.eslintrc.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	extends: ['./node_modules/@balena/lint/config/.eslintrc.js'],
+	parserOptions: {
+		project: 'tsconfig.js.json',
+		sourceType: 'module',
+	},
+	env: {
+		jest: true,
+	},
+	root: true,
```

Comment:
What does this do?

Reply:
`root: true` is not needed as no other eslintrc needs to be loaded. Deleted.